thumbor/handlers/__init__.py

```diff
@@ -620,6 +620,8 @@ def _fetch(self, url):
             is_mixed_no_file_storage = is_mixed_storage and isinstance(storage.file_storage, NoStorage)
 
             if not (is_no_storage or is_mixed_no_file_storage):
+                self.context.config = copy.deepcopy(self.context.config)
+                self.context.config.PRESERVE_EXIF_INFO = True
```

Comment:
probably, but this would be a bigger change and I'm fine if someone is having the overview and/or time to do this. In the meanwhile I think it is absolutely valid to fix this bug in such a way (we actually thought about passing the 'strip exif' attribute to the engine but this would change a lot)